frontend/src/index.js

```diff
@@ -1,9 +1,31 @@
+/* eslint-disable prettier/prettier */
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { App } from './components';
+import './index.css';
+import App from './App';
+import thunk from 'redux-thunk';
+import logger from 'redux-logger';
+import { Provider } from 'react-redux';
+import { createStore, applyMiddleware } from 'redux';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { userReducers } from './reducers';
```

Comment:
userReducers is under src/store/reducers/. I'd also suggest breaking up the user and article stuff into separate reducers before the reducer files begin to sprawl too much..